server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -1073,6 +1078,12 @@ def set_credentials(
                 or mlrun.mlconf.model_endpoint_monitoring.stream_connection
                 or _default_secrets_v3io
             )
+
+        if stream_profile_name:
+            # TODO: Add checks.
```

Comment:
?

Reply:
Once Kafka is also in, this step will validate that the given profile name is V3IO/Kafka.